packages/notifi-react-card/README.md

```diff
@@ -11,6 +11,32 @@ npm install --save-dev @notifi-network/notifi-core
 
 You can import the default stylesheet to get baseline styling.
 
+## Video
+
+Below is a video useful for setting up the Notifi React Card.
+
+<iframe
+    width="640"
+    height="480"
+    src="https://www.youtube.com/embed/LIKu-clf9bg"
+    frameborder="0"
+    allow="autoplay; encrypted-media"
+    allowfullscreen
+>
+</iframe>
+
+### Design Guidelines
+
+We have general guidelines that we like to follow. You can see them here: [Figma](https://www.figma.com/file/ieF0Ynuc3WI608RCt7wKSf/Notifi-Template?node-id=0%3A1&t=v8zeo6UovJAOb9vR-0).
+
```

Comment:
Suggestion: Should we show a final screenshot? Like an example of a partner site we really like? 

I have a feeling that people either won't look at the Figma or might have a hard time visualizing it without an example site or final screenshot of a live site.

Reply:
done